### PR TITLE
[STUD-28] Make a CatGenerator

### PR DESCRIPTION
**Pull request by MikhailSkrebcov**
trello.com/c/VSoRLdnD

### CODE REVIEW

File: src/main/java/com/stud/java/service/generator/CatGenerator.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.UUID;
+import java.util.ArrayList;
+import java.util.Random;
+
+
+public class CatGenerator extends CatEntity {
```
Comment (VladSKR) [inline src/main/java/com/stud/java/service/generator/CatGenerator.java]: Why extends???

File: src/main/java/com/stud/java/service/generator/CatGenerator.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    private static final List<String> NAMES = List.of("Муся", "Томас", "Тоша", "Маруся", "Жека", "Красавчик", "Васаби", "Суши", "Стив", "Стелла", "Кирби", "Марио-Сан", "Тайбой", "Блейд", "Санбой");
+    private static Random Random = new Random();
+
+    public static UUID generateId() {
+        return UUID.randomUUID();
+    }
```
Comment (VladSKR) [inline src/main/java/com/stud/java/service/generator/CatGenerator.java]: You dont need to extract this to private method, just use UUID.randomUUID() wherever you need

File: src/main/java/com/stud/java/service/generator/CatGenerator.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+    public static String generateName() {
```
Comment (VladSKR) [inline src/main/java/com/stud/java/service/generator/CatGenerator.java]: Why public?

File: src/main/java/com/stud/java/service/generator/CatGenerator.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        return NAMES.get(Random.nextInt(NAMES.size()));
+    }
+
+    public static LocalDate generateBirthDate() {
```
Comment (VladSKR) [inline src/main/java/com/stud/java/service/generator/CatGenerator.java]: why public?

**Comment by VladSKR** [comment]
You need to rename branch to feature/STUD-28

**Comment by MikhailSkrebcov** [comment]
> You need to rename branch to feature/STUD-28

image

**Pull request closed without merging**